### ISSUE TITLE
How can we manage a Changelog better?

### ISSUE THREAD

**Issue by edeutsch** [RTXteam/RTX#2150]
So it seems to becoming standard that when I request a deployment to ITRB, they want a Changelog.

How can we best manage a shared Changelog so that I don't need to snuffle around commit messages and make something up when it comes time to make the request?

Should we have a Google doc where everyone tries to remember to make a useful Changelog entry (primarily for Sarah's consumption)?
Or try to track them in an issue? or a Discission?

Basically, the end result is that when I make a deployment request to ITRB, I'd like to be able to do a copy-paste from a Changelog document, rather than try to piece things together when Sarah asks.

Ideas? suggestions?

**Comment by saramsey** [comment]
FWIW, for RTX-KG2, we maintain a Markdown file that documents each release:

https://github.com/RTXteam/RTX-KG2/blob/master/kg2-versions.md

We mostly just reference GitHub issues to document the changelog.

**Comment by edeutsch** [comment]
The goal would be that I have a relatively high level summary of major changes ready to go when I need to put in the deployment request. Only important changes need to be listed, we don't need "fix typo", "removed unused line", "quiet debug messages". And ideally I wouldn't have to wade through loads of commits trying to decide what's important and what's trivial, etc.

A terse list that we can all edit would be nice. I'm thinking doing that through GitHub might be tedious. Google doc might be easiest. We also need to figure out out how to be diligent about keeping it up.

I suggest we discuss at AHM.

**Comment by edeutsch** [comment]
At AHM we realized that we can edit other peoples comments on GH, and so we decided to just have one issue per deployment interval. The first one is here: #2154 
Please add to that whenever you make a change that NCATS might care about, or seems substantial enough that announcing it is useful.
closing this.
